Source/QED/breit_wheeler_engine_wrapper.h

```diff
@@ -0,0 +1,27 @@
+#ifndef WARPX_breit_wheeler_engine_wrapper_h_
+#define WARPX_breit_wheeler_engine_wrapper_h_
+
+//This file provides a wrapper aroud the breit_wheeler engine
+//provided by the standard template library
+
+//BW ENGINE
+#include "qed_wrapper_commons.h"
+#include "breit_wheeler_engine.hpp"
+
+#include "amrex_rng_wrapper.h"
+
+class warpx_breit_wheeler_engine :
+    public picsar::multi_physics::
+    breit_wheeler_engine<amrex::Real, amrex_rng_wrapper>
```

Comment:
The idea is that I would like to be able to add (maybe radical) changes to the QED library without affecting too much the WarpX code. For example, if in a few months we switch to a template-based solution to choose between SI units and normalized units, using a wrapper would allow us to change just a couple of files in the WarpX code. 

Instead of creating a derived class, I could put just pure functions  in the QED folder  (sorted in different files). For example, a warpx_breit_wheeler_engine.hpp file could contain:
- definition of PXRMP_GPU & PXRMP_WITH_SI_UNITS
- inclusion of the relevant files of the QED library
- pure functions wrapping the calls to the static functions of the QED libraries
- a pure function which initializes the lookup tables instantiating a BW object

what do you think?